src/test/java/com/conveyal/datatools/manager/jobs/MergeFeedsJobTest.java

```diff
@@ -0,0 +1,181 @@
+        File feed = FeedVersion.feedStore.getFeed(project.id + ".zip");
+        FeedVersion mergedVersion = createFeedVersion(source, feed);
+        // Ensure the feed has the row counts we expect.
+        assertEquals(
+            "trips count for merged feed should equal sum of trips for versions merged.",
+            bartVersion1.feedLoadResult.trips.rowCount + calTrainVersion.feedLoadResult.trips.rowCount,
+            mergedVersion.feedLoadResult.trips.rowCount
+        );
+        assertEquals(
+            "routes count for merged feed should equal sum of routes for versions merged.",
+            bartVersion1.feedLoadResult.routes.rowCount + calTrainVersion.feedLoadResult.routes.rowCount,
+            mergedVersion.feedLoadResult.routes.rowCount
+            );
+        assertEquals(
+            "stops count for merged feed should equal sum of stops for versions merged.",
+            mergedVersion.feedLoadResult.stops.rowCount,
+            bartVersion1.feedLoadResult.stops.rowCount + calTrainVersion.feedLoadResult.stops.rowCount
+        );
+        assertEquals(
+            "agency count for merged feed should equal sum of agency for versions merged.",
+            mergedVersion.feedLoadResult.agency.rowCount,
+            bartVersion1.feedLoadResult.agency.rowCount + calTrainVersion.feedLoadResult.agency.rowCount
+        );
+        assertEquals(
+            "stopTimes count for merged feed should equal sum of stopTimes for versions merged.",
+            mergedVersion.feedLoadResult.stopTimes.rowCount,
+            bartVersion1.feedLoadResult.stopTimes.rowCount + calTrainVersion.feedLoadResult.stopTimes.rowCount
+        );
+        assertEquals(
+            "calendar count for merged feed should equal sum of calendar for versions merged.",
+            mergedVersion.feedLoadResult.calendar.rowCount,
+            bartVersion1.feedLoadResult.calendar.rowCount + calTrainVersion.feedLoadResult.calendar.rowCount
+        );
+        assertEquals(
+            "calendarDates count for merged feed should equal sum of calendarDates for versions merged.",
+            mergedVersion.feedLoadResult.calendarDates.rowCount,
+            bartVersion1.feedLoadResult.calendarDates.rowCount + calTrainVersion.feedLoadResult.calendarDates.rowCount
+        );
+    }
+
+    /**
+     * Ensures that an MTC merge of feeds with duplicate trip IDs will fail.
+     */
+    @Test
+    public void mergeMTCShouldFailOnDuplicateTrip() {
+        Set<FeedVersion> versions = new HashSet<>();
+        versions.add(bartVersion1);
+        versions.add(bartVersion2);
+        MergeFeedsJob mergeFeedsJob = new MergeFeedsJob("test", versions, "merged_output", MergeFeedsType.MTC);
+        // Run the job in this thread (we're not concerned about concurrency here).
+        mergeFeedsJob.run();
+        // Result should fail.
+        assertEquals(
+            "Merge feeds job should fail due to duplicate trip IDs.",
```

Comment:
Just so I know, is the logic here... if two different feeds have a trip with the same trip ID, then the merge should fail?

Reply:
I suppose the next test answers my question, ha!

Reply:
Yes, that's correct!